projects/rich-previews/src/google-product-desktop/google-product-desktop.css

```diff
@@ -0,0 +1,260 @@
+@import "../../@shared/css/defaults.css";
+@import "../../@shared/css/google.css";
+
+.scroll-container--fake {
+  max-width: 70.25rem;
+}
+
+/* Product */
+.product {
+  background-color: var(--color-white);
+  border-radius: 0.5rem;
+  box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.16);
+  max-width: 70.25rem;
+  min-width: 38.875rem;
+  padding: 0.75rem;
+}
+
+.product--no-hover {
+  cursor: default;
+}
+
+.product__image,
+.product__merchant-name,
+.product__merchant-review,
+.product__name,
+.product__review {
+  cursor: pointer;
+}
+
+.product__merchant-name:hover,
+.product__merchant-review:hover,
+.product__name:hover,
+.product__review:hover,
+.product-carrousel__item:hover .product__name {
+  text-decoration: underline;
+}
+
+.product__image {
+  display: block;
+  height: 11.25rem;
+  margin: 0.75rem 0.75rem 0.75rem 0;
+  object-fit: contain;
+  object-position: center;
+  width: 11.25rem;
+}
+
+.product__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0.75rem;
+}
+
+.product__merchant-name,
+.product__merchant-review,
+.product__price {
+  line-height: 1.75rem;
+  margin-right: 0.375rem;
+}
+
+.product__name {
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  color: var(--color-shuttle-gray);
+  display: -webkit-box;
```

Comment:
I assume these style come from Google directly? Of not, what does this do and how good is the support in other browsers?

Reply:
Ja, dit komt van Google, en hoewel er `-webkit-` staat, werkt het ook in Firefox.